README.md

```diff
@@ -43,6 +43,8 @@ Instrument speedtest.net speedtests from Prometheus.
 
 optional arguments:
   -h, --help            show this help message and exit
+  -a address, --address address
```

Comment:
You missed [-a address] after [-h] in the "usage:" line

usage: prometheus_speedtest.py [-h] [-a address] [-p port] [-v]